doc/man1/dgst.pod

```diff
@@ -8,7 +8,7 @@ dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5, blak
 
 B<openssl> B<dgst>
 [B<-help>]
-[B<-I<digest>>]
+[B<-[digest]>]
```

Comment:
On side note, this page is misleading in sense that it lists sha and blake2[bs] as standalone commands. These don't work. It might be appropriate to add a note that those that are provided for backward compatibility and should not be used in new scripts. BTW, it refers to list --digest-commands, which seems to be misleading at least on gost count. I mean if you issue `apps/openssl gost apps/openssl` it will emit SHA256 hash value...